app/services/experimentation/runner.rb

```diff
@@ -41,25 +53,31 @@ def start_current_patient_experiment(name:, percentage_of_patients: 100)
         end
       end
 
+      logger.info("Finished scheduling notifications for #{name} experiment - marking as running and exiting.")
       experiment.running_state!
     end
 
-    def schedule_daily_stale_patient_notifications(name:, patients_per_day: PATIENTS_PER_DAY)
-      experiment = Experimentation::Experiment.find_by(name: name, experiment_type: "stale_patients", state: [:new, :running])
-      unless experiment
-        Sentry.capture_message("Experiment #{name} not found and may need to be removed from scheduler.")
+    def self.schedule_daily_stale_patient_notifications(name:, patients_per_day: PATIENTS_PER_DAY)
```

Comment:
Changing to `self.` allows rails constant lookup to just work when we are inside a module like this